paddle/CMakeLists.txt

```diff
@@ -17,11 +17,14 @@ add_subdirectory(strings)
 
 find_package(Boost QUIET)
 
-if(Boost_FOUND)
-  include_directories(${Boost_INCLUDE_DIRS})
-  include_directories(${CMAKE_CURRENT_SOURCE_DIR})
-  add_subdirectory(majel)
-endif()
+add_subdirectory(platform)
+add_subdirectory(framework)
+
+# if(Boost_FOUND)
+#   include_directories(${Boost_INCLUDE_DIRS})
```

Comment:
I got it. Let me add the check of the existence of Boost. Thanks for the remind!